packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -1409,14 +1409,24 @@ class _TextSelectionGestureDetectorState extends State<TextSelectionGestureDetec
   DragStartDetails? _lastDragStartDetails;
   DragUpdateDetails? _lastDragUpdateDetails;
   Timer? _dragUpdateThrottleTimer;
+  bool _isMouseDragging = false;
```

Comment:
HorizontalDragGestureRecognizer allows you to specify the type of pointer to accept, but now that we've switched to PanGestureRecognizer, which doesn't support that, we have to manually make sure we only listen to mouse drags.

Reply:
Would it be possible to add that to the existing `PanGestureRecognizer` or have a private subclass of `PanGestureRecognizer`?

Reply:
Great idea, looks like it's actually only a 2 line change to add it to PanGesureRecognizer because the super class already supports it.